modules/uhsg_feedback/uhsg_feedback.module

```diff
@@ -32,7 +35,12 @@ function uhsg_feedback_mail_alter(&$message) {
     $message['body'] = [];
 
     // Subject.
-    $message['subject'] = t('Feedback from Instructions for students');
+    // TODO: Different text for teaching instructions.
+    if(\Drupal::service('uhsg_domain.domain')->isTeachingDomain()){
+      $message['subject'] = t('Feedback from Instructions for teaching');
+    }else{
+      $message['subject'] = t('Feedback from Instructions for students');
+    }
```

Comment:
Check coding standards (https://www.drupal.org/docs/develop/standards/coding-standards):

- Use whitespace `if (` ... `) {`
- else should start on new line

Reply:
I had not correctly activated Atom/linter/drupalcs for coding standards in this project. 
Should be fixed now.